.gitignore

```diff
@@ -39,5 +39,15 @@ gtda/externals/pybind11
 .pytest_cache/
 .hypothesis/
 
+# Latex
```

Comment:
A more future-proof solution would be to ignore *all* .aux, .bbl, .blg, .brf, etc. files from anywhere. You can use the wildcard * as in e.g. *.aux, as is done a few lines above for "Output files"

Reply:
OK, I was just worried about some extensions like .log

Will * them all